Source/Core/DiscIO/WbfsBlob.h

```diff
@@ -31,23 +33,28 @@ class WbfsFileReader : public IBlobReader
   bool Read(u64 offset, u64 nbytes, u8* out_ptr) override;
 
 private:
-  WbfsFileReader(const std::string& filename);
+  WbfsFileReader(File::IOFile&& file, const std::string& path);
 
-  bool OpenFiles(const std::string& filename);
+  void OpenAdditionalFiles(const std::string& path);
+  bool AddFileToList(File::IOFile&& file);
   bool ReadHeader();
 
   File::IOFile& SeekToCluster(u64 offset, u64* available);
   bool IsGood() { return m_good; }
   struct file_entry
   {
+    file_entry(File::IOFile&& file, u64 base_address, u64 size)
```

Comment:
this will cause variable shadowing warnings on Linux if the constructor params don't have suffixed underscores.